core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -1360,6 +1360,14 @@ class KafkaConfig private(doLog: Boolean, val props: java.util.Map[_, _], dynami
       }
       validateAdvertisedListenersNonEmptyForBroker()
     }
+    if (processRoles.contains(ProcessRole.BrokerRole)
+      && originals.containsKey(ReplicationConfigs.INTER_BROKER_PROTOCOL_VERSION_CONFIG)
+      && logDirs.size > 1) {
+        require(interBrokerProtocolVersion.isDirectoryAssignmentSupported,
+          s"Multiple log directories (aka JBOD) are not supported in the configured " +
+            s"${ReplicationConfigs.INTER_BROKER_PROTOCOL_VERSION_CONFIG} ${interBrokerProtocolVersion}. " +
```

Comment:
Should we add `ReplicationConfigs.INTER_BROKER_PROTOCOL_VERSION_CONFIG` after the value? So the message looks like `Multiple log directories (aka JBOD) are not supported with the configured 3.6 inter.broker.protocol.version`